diff --git a/public/assets/js/app.js b/public/assets/js/app.js
index b650266..8ee2c45 100644
--- a/public/assets/js/app.js
+++ b/public/assets/js/app.js
@@ -2,7 +2,6 @@
 
 const $ = require('jquery');
 require('bootstrap');
-global.jQuery = $;
 require('jquery-tags-input/dist/jquery.tagsinput.min');
 
     $(document).ready(function() {
diff --git a/webpack.config.js b/webpack.config.js
index 2ce4c7d..b0fb8a1 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -46,7 +46,7 @@ Encore
     //.enableTypeScriptLoader()
 
     // uncomment if you're having problems with a jQuery plugin
-    //.autoProvidejQuery()
+    .autoProvidejQuery()
 ;
 
 module.exports = Encore.getWebpackConfig();
