diff --git a/assets/js/login.js b/assets/js/login.js
index 1cfd3dd..a7243c0 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -20,7 +20,7 @@ import '../css/login.css';
 
             if ($emailInput.val().length >= 30) {
                 import('./Components/show_long_email_error').then(showLoginEmailError => {
-                    showLoginEmailError($emailInput);
+                    showLoginEmailError.default($emailInput);
                 });
             }
         });
