diff --git a/public/assets/js/app.js b/public/assets/js/app.js
index b650266..b89a57a 100644
--- a/public/assets/js/app.js
+++ b/public/assets/js/app.js
@@ -4,6 +4,7 @@ const $ = require('jquery');
 require('bootstrap');
 global.jQuery = $;
 require('jquery-tags-input/dist/jquery.tagsinput.min');
+require('../css/app.css');
 
     $(document).ready(function() {
         $('[data-toggle="tooltip"]').tooltip();
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 612a630..c886893 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -11,7 +11,8 @@
     {% block stylesheets %}
         <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
         <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
-        <link href="{{ asset('assets/css/app.css') }}" rel="stylesheet" />
+        {{ encore_entry_link_tags('app') }}
+
     {% endblock %}
 
     <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
