diff --git a/assets/js/Components/Routing.js b/assets/js/Components/Routing.js
index 511d98d..f129d44 100644
--- a/assets/js/Components/Routing.js
+++ b/assets/js/Components/Routing.js
@@ -3,6 +3,7 @@ import Routing from '../../../vendor/friendsofsymfony/jsrouting-bundle/Resources
 
 Routing.setRoutingData(routes);
 
+console.log(IS_PRODUCTION);
 fetch('/js/routing.json').then(Response => {
     Response.json().then(data => {
         Routing.setRoutingData(routes);
diff --git a/webpack.config.js b/webpack.config.js
index 05d3496..8e4325c 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -54,6 +54,9 @@ Encore
 
     // uncomment if you're having problems with a jQuery plugin
     .autoProvidejQuery()
+    .configureDefinePlugin((options) => {
+        options.IS_PRODUCTION = JSON.stringify(Encore.isProduction());
+    })
 ;
 
 module.exports = Encore.getWebpackConfig();
