diff --git a/assets/js/api/rep_log.js b/assets/js/api/rep_log.js
index 4a8d297..33ae342 100644
--- a/assets/js/api/rep_log.js
+++ b/assets/js/api/rep_log.js
@@ -1,7 +1,10 @@
 import Routing from '../Components/Routing';
 
 export function getRepLogs() {
-    return fetch(Routing.generate('rep_log_list'))
+    return fetch(Routing.generate('rep_log_list'), {
+        // is the new default value
+        credentials: 'same-origin'
+    })
         .then(response => {
             return response.json();
         });
