diff --git a/assets/js/Components/Routing.js b/assets/js/Components/Routing.js
index dfe4d01..dd3861f 100644
--- a/assets/js/Components/Routing.js
+++ b/assets/js/Components/Routing.js
@@ -3,4 +3,8 @@ import Routing from '../../../vendor/friendsofsymfony/jsrouting-bundle/Resources
 
 Routing.setRoutingData(routes);
 
+fetch('/js/routing.json').then(Response => {
+    console.log(Response.json());
+})
+
 export default Routing;
