diff --git a/public/assets/js/RepLogApp.js b/public/assets/js/RepLogApp.js
index 2204c31..ddcfb1f 100644
--- a/public/assets/js/RepLogApp.js
+++ b/public/assets/js/RepLogApp.js
@@ -2,8 +2,6 @@
 
 const Helper = require('./RepLogAppHelper');
 
-(function(window, $, Routing, swal) {
-
     let HelperInstances = new WeakMap();
 
     class RepLogApp {
@@ -212,5 +210,4 @@ const Helper = require('./RepLogAppHelper');
 </tr>
 `;
 
-    window.RepLogApp = RepLogApp;
-})(window, jQuery, Routing, swal);
+module.exports = RepLogApp;
