diff --git a/public/assets/js/rep_log.js b/public/assets/js/rep_log.js
new file mode 100644
index 0000000..a42e5e7
--- /dev/null
+++ b/public/assets/js/rep_log.js
@@ -0,0 +1,6 @@
+const RepLogApp = require('./RepLogApp');
+
+$(document).ready(function() {
+    var $wrapper = $('.js-rep-log-table');
+    var repLogApp = new RepLogApp($wrapper);
+});
diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index 585d1bf..d2a4c7f 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -56,11 +56,4 @@
 
     <script src="https://cdn.jsdelivr.net/sweetalert2/6.1.0/sweetalert2.min.js"></script>
     <script src="{{ asset('build/rep_log.js') }}"></script>
-
-    <script>
-        $(document).ready(function() {
-            var $wrapper = $('.js-rep-log-table');
-            var repLogApp = new RepLogApp($wrapper);
-        });
-    </script>
 {% endblock %}
diff --git a/webpack.config.js b/webpack.config.js
index bac90b7..320b317 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -17,7 +17,7 @@ Encore
      * Each entry will result in one JavaScript file (e.g. app.js)
      * and one CSS file (e.g. app.css) if you JavaScript imports CSS.
      */
-    .addEntry('rep_log', './public/assets/js/RepLogApp.js')
+    .addEntry('rep_log', './public/assets/js/rep_log.js')
     //.addEntry('page1', './assets/js/page1.js')
     //.addEntry('page2', './assets/js/page2.js')
 
