diff --git a/public/assets/js/login.js b/public/assets/js/login.js
index b077104..9bcf3be 100644
--- a/public/assets/js/login.js
+++ b/public/assets/js/login.js
@@ -1,6 +1,7 @@
 'use strict';
 
-(function(window, $) {
+const $ = require('jquery');
+
     $(document).ready(function() {
         $('.js-recommended-login').on('click', '.js-show-login', function(e) {
             e.preventDefault();
@@ -19,4 +20,3 @@
             }
         });
     });
-})(window, jQuery);
