diff --git a/assets/images/dumbbell.png b/assets/static/dumbbell.png
similarity index 100%
rename from assets/images/dumbbell.png
rename to assets/static/dumbbell.png
diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index 41ec3ff..473d6bf 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -35,7 +35,7 @@
         <div class="col-md-5">
             <div class="leaderboard">
                 <h2 class="text-center">
-                    <img class="dumbbell" src="{{ asset('assets/images/dumbbell.png') }}" />
+                    <img class="dumbbell" src="{{ asset('build/dumbbell.png') }}" />
                     Leaderboard
                 </h2>
 
diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index ebbebcc..474c63c 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -18,7 +18,7 @@
     <div class="container">
         <div class="wrapper">
             <form method="post" class="form-signin">
-                <h3><img class="dumbbell" src="{{ asset('assets/images/dumbbell.png') }}">Login! Start Lifting!</h3>
+                <h3><img class="dumbbell" src="{{ asset('build/dumbbell.png') }}" />Login! Start Lifting!</h3>
                 <div class="js-recommended-login">
                     <a href="#" class="js-show-login" title="Click for login details">Don't know the login?</a>
                     <div class="js-recommended-login-details" style="display: none;">
diff --git a/webpack.config.js b/webpack.config.js
index 1a03562..b40f630 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -26,6 +26,10 @@ Encore
     // but, you probably want this, unless you're building a single-page app
     .disableSingleRuntimeChunk()
 
+    .copyFiles([{
+        from: './assets/static'
+    }])
+
     .splitEntryChunks()
 
     /*
