diff --git a/conference.js b/conference.js
index 236b230..4ee5028 100644
--- a/conference.js
+++ b/conference.js
@@ -1,9 +1,9 @@
 module.exports = class {
     constructor(name) {
-        this.name = name;
+        this._name = name;
     }
 
-    getName() {
-        return this.name;
+    get name() {
+        return this._name;
     }
 };
diff --git a/play.js b/play.js
index 25054f0..6a20b53 100644
--- a/play.js
+++ b/play.js
@@ -2,4 +2,4 @@ const Conference = require('./conference');
 
 const sfCon = new Conference('SymfonyCon 2018');
 
-console.log(sfCon.getName());
+console.log(sfCon.name);
