diff --git a/play.js b/play.js
index d4e6ebb..cee22da 100644
--- a/play.js
+++ b/play.js
@@ -7,7 +7,11 @@ sfCon.printNameLater()
         console.log('callback!');
         console.log(data);
 
-        return data * 100;
+        return new Promise((resolve, reject) => {
+            setTimeout(() => {
+                resolve(data * 100);
+            }, 2000)
+        });
     })
     .then((data) => {
         console.log(data);
