diff --git a/conference.js b/conference.js
index 6e7e5b0..de789c8 100644
--- a/conference.js
+++ b/conference.js
@@ -8,8 +8,11 @@ module.exports = class {
     }
 
     printNameLater() {
-        setTimeout(() => {
-            console.log(this.name);
-        }, 500);
+        return new Promise((resolve, reject) => {
+            setTimeout(() => {
+                console.log(this.name);
+                resolve();
+            }, 500);
+        });
     }
 };
diff --git a/play.js b/play.js
index ddf49e4..1df0ccd 100644
--- a/play.js
+++ b/play.js
@@ -2,5 +2,8 @@ const Conference = require('./conference');
 
 const sfCon = new Conference('SymfonyCon 2018');
 
-sfCon.printNameLater();
+sfCon.printNameLater()
+    .then(() => {
+        console.log('callback!');
+    });
 console.log('Done!');
