diff --git a/conference.js b/conference.js
index de789c8..67bdef7 100644
--- a/conference.js
+++ b/conference.js
@@ -11,7 +11,7 @@ module.exports = class {
         return new Promise((resolve, reject) => {
             setTimeout(() => {
                 console.log(this.name);
-                resolve();
+                resolve(Math.random());
             }, 500);
         });
     }
diff --git a/play.js b/play.js
index 1df0ccd..aa6bc4a 100644
--- a/play.js
+++ b/play.js
@@ -3,7 +3,8 @@ const Conference = require('./conference');
 const sfCon = new Conference('SymfonyCon 2018');
 
 sfCon.printNameLater()
-    .then(() => {
+    .then((data) => {
         console.log('callback!');
+        console.log(data);
     });
 console.log('Done!');
