diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index 739833b..585d1bf 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -55,7 +55,7 @@
     {{ parent() }}
 
     <script src="https://cdn.jsdelivr.net/sweetalert2/6.1.0/sweetalert2.min.js"></script>
-    <script src="{{ asset('assets/js/RepLogApp.js') }}"></script>
+    <script src="{{ asset('build/rep_log.js') }}"></script>
 
     <script>
         $(document).ready(function() {
